src/run.rs

```diff
@@ -187,7 +186,12 @@ pub fn run() {
     .collect::<Vec<&str>>();
 
   let justfile_option = matches.value_of("JUSTFILE");
-  let working_directory_option = matches.value_of("WORKING-DIRECTORY");
+  let mut working_directory_option = matches.value_of("WORKING-DIRECTORY");
```

Comment:
Let's change `working_directory_option` to a `Option<&Path>`, so that we're working with paths from here on out, and don't have to convert to a string. You can do `.map(Path::new)`.

Reply:
Done.